examples/chart/teleport-cluster/templates/psp.yaml

```diff
@@ -38,6 +38,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   name: {{ .Release.Name }}-psp
+  namespace: {{ .Release.Namespace }}
```

Comment:
Did we decide that PSPs could be namespaced in the end? I thought the idea was that they shouldn't and this wasn't a bug after all.

Reply:
It's complicated... PSPs are not namespaced, but their role and role bindings are. This resource is a bit awkward, and I'm happy with it being phased out. This makes me think that we'll need to test against the new built-in security policy thing and see if we need to add labels/annotations to get enrolled in the restricted policy.